storage/innobase/xtrabackup/src/xtrabackup.h

```diff
@@ -331,4 +331,9 @@ bool xb_process_datadir(const char *path,   /*!<in: datadir path */
                         void *data); /*!<in: additional argument for
                                      callback */
 
+/* update the checkpoint and checksum in both log header buffer
```

Comment:
Update the checkpoint LSN and recalcuate the checksum of log header

Reply:
fixed